### DISCUSSION TITLE
How can I install this on CentOS ?

### DISCUSSION THREAD

**Discussion by slacklife** [bashly-framework/bashly#254] [Q&A]
How can I install this on CentOS ?

I cloned it on CentOS and run `./bashly/bin/bashly --init`.
I got that
```
/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bashly (LoadError)
	from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require'
	from ./bashly/bin/bashly:2:in `<main>'
```

**Comment by DannyBen** [comment]
This is not how you install things.

You have two installation options:

1. If you have Ruby version 2.7 at least, you can run `gem install bashly`
2. If not, and you have docker, you can create a docker alias.

Installation instructions are here.
https://bashly.dannyb.co/installation/

There is no need to clone the repo.
